signer/src/lib.rs

```diff
@@ -33,6 +33,7 @@ use secp256k1::util::{
     COMPRESSED_PUBLIC_KEY_SIZE, FULL_PUBLIC_KEY_SIZE, SECRET_KEY_SIZE, SIGNATURE_SIZE,
 };
 use secp256k1::{recover, sign, verify, Message, RecoveryId};
+use zeroize::Zeroize;
```

Comment:
Can we split this PR and get the Zeroize into another?
We need to share a link per fix in the report

Reply:
Done !!. Thinking of other alternatives or implications to use zeroize on structs that are shared through the wasm interface to javascript.